Python/susankoju/task2/scraper.py

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/python3
+import os
+import csv
+import json
+import yaml
+import sqlite3
+import argparse
+import requests
+from bs4 import BeautifulSoup
+import xml.etree.ElementTree as ET
+
+DEFAULT_MAX = 50
+OUTPUT_FOLDER = 'output'
+DEFAULT_SEARCH = 'computer'
+DEFAULT_FILENAME = 'results'
+BASE_URL = 'https://hamrobazaar.com/'
+BASE_SEARCH_URL = BASE_URL + 'search.php'
+
+
+def get_search_result_rows(data, max=DEFAULT_MAX):
+    ''' returns result items array '''
+    rows = []
+    done = 0
+    tables = data.find_all('table', attrs={'border':"0", 'width':"100%", 'cellspacing':"0", 'cellpadding':"0"})
+    for table in tables:
+        if table.find('td', attrs={'bgcolor':'#C6C6D9'}):
+            td_data = table.find_all('td', attrs={'bgcolor': '#F2F4F9'})
+            if not td_data:
+                # Alternating result background color
+                td_data = table.find_all('td', attrs={'bgcolor': '#ECF0F6'})
+            if td_data:
+                rows.append(td_data)
+                done += 1
+                if done >= max: break
+
+    return rows
+
+
+def store_data_in_db(json_data, header, filename=DEFAULT_FILENAME):
+    ''' Store data in DB '''
+    print('\nStoring data in SQLite database...')
+    conn = sqlite3.connect(f'{OUTPUT_FOLDER}/{filename}.db')
+    cursor = conn.cursor()
```

Comment:
You can use context manager here
with con:
con.execute(
Reference:
https://docs.python.org/3/library/sqlite3.html#using-the-connection-as-a-context-manager